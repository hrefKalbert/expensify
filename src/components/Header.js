import React from 'react'
import {NavLink} from 'react-router-dom'


const Header = () =>
  <header>
    <h1>Expensify</h1>
    <NavLink activeClassName="is-active" to="/" exact={true}>Dash board</NavLink>
    <NavLink activeClassName="is-active" to="/create">Go to Create page</NavLink>
  </header>
 

export default Header