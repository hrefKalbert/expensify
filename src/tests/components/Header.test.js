import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'
import ReactShallowRenderer from 'react-test-renderer/shallow'
import Header from '../../components/Header'

test('shoud render Header correctly', () => {
  const wrapper = shallow(<Header />)
  expect(wrapper).toMatchSnapshot()
  
  // const renderer = new ReactShallowRenderer()
  // renderer.render(<Header />)
  // console.log(renderer.getRenderOutput())
  // expect(renderer.getRenderOutput()).toMatchSnapshot()
})