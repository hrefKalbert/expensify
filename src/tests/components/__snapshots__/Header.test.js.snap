// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shoud render Header correctly 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Dash board
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/create"
  >
    Go to Create page
  </NavLink>
</header>
`;
